infrastructure/application/services/hasura.ts

```diff
@@ -34,10 +34,6 @@ export const createHasuraService = async ({
     protocol: "HTTP",
     healthCheck: {
       path: "/healthz",
-      interval: 30,
```

Comment:
The defaults for the ALB health checking the Hasura service are reasonable, so no need for this.